sycl/include/CL/sycl/queue.hpp

```diff
@@ -236,7 +236,11 @@ class __SYCL_EXPORT queue {
 #ifdef DISABLE_SYCL_INSTRUMENTATION_METADATA
     const detail::code_location &CodeLoc = {};
```

Comment:
This definition isn't needed after your change. The same is true in the other submit_barrier case.

On the other hand, it would be nice to make all the code in this header consistent. The _CODELOCPARAM() and _CODELOCARG() macros later in the file address this is a different way.

Reply:
> This definition isn't needed after your change. The same is true in the other submit_barrier case.

Nice catch. Done.

> On the other hand, it would be nice to make all the code in this header consistent. The _CODELOCPARAM() and _CODELOCARG() macros later in the file address this is a different way.

I prefer this to be handled in distinct patch.